mlflow/server/js/src/gateway/types.ts

```diff
@@ -6,6 +6,15 @@ export interface Model {
   model: string;
   provider: string;
   supports_function_calling: boolean;
+  supports_vision?: boolean;
```

Comment:
Can we call this interface as "ProviderModel" to disambiguate?

Reply:
yep!